docs/competition/team-support/helpdesk.md

```diff
@@ -45,27 +43,27 @@ Whenever a team comes to Helpdesk with an issue that cannot be solved instantly
 
 ### Handling of requests for robot safety checks
 
-The robots require safety checking before they are allowed to compete in the competition. Teams will not be given their hi-viz vest until their robot has passed the safety check procedure. Helpdesk volunteers should direct teams enquiring about safety checks to the Health & Safety Coordinator (Scarzy).
+The robots require safety checking before they are allowed to compete in the competition. Teams will not be given their hi-viz vest until their robot has passed the safety check procedure. Helpdesk volunteers should direct teams enquiring about safety checks to the Health & Safety Coordinator.
 
 ### Handling of general robot development requests
 
 Many teams will come to Helpdesk requesting help with the development of their robot or, more likely, help using the Student Robotics kit and associated software. As mentioned earlier, Helpdesk volunteers are not expected to tell teams what to do, but rather offer advice. Helpdesk volunteers should strive to answer questions about the use of the kit and software.
 
 Some queries may require a volunteer to visit the team in their pit to observe behaviour in situ. This must only be done by a Helpdesk volunteer if it will leave no fewer than two volunteers actively manning Helpdesk. If it is not possible for a Helpdesk volunteer to visit a team then a [Roving Helper](/competition/team-support/Roving_Helper) should be summoned, via radio if necessary.
 
-In some rare situations it may be necessary to request the assistance of a person with more in-depth knowledge of the kit to help solve an issue. This should only be done if the issue is non-trivial. The following people are available for specific help and can be summoned over the radio:
+In some rare situations it may be necessary to request the assistance of a person with more in-depth knowledge of the kit to help solve an issue. This should only be done if the issue is non-trivial, Here is a list of people to contact for further help:
 
-Electronic Hardware
-:   Rich Barlow
-:   Rob Spanton
+List for SR2019:
```

Comment:
This list should definitely be significantly longer than this

Reply:
Why? There's always someone who is a 5/5 on knowing the kit at helpdesk, so this is for if nobody has a clue, in which case it's the people who wrote/made those respective systems.

Reply:
ah, hmm that's a fair point. I can live with it.